questionable.gemspec

```diff
@@ -1,33 +1,31 @@
-$:.push File.expand_path("../lib", __FILE__)
+$:.push File.expand_path('../lib', __FILE__)
 
-# Maintain your gem's version:
-require "questionable/version"
+require 'questionable/version'
 
-# Describe your gem and declare its dependencies:
 Gem::Specification.new do |s|
-  s.name        = "questionable_surveys"
+  s.name        = 'questionable_surveys'
   s.version     = Questionable::VERSION
```

Comment:
If I can find a better way to bootstrap tests (like https://github.com/pat/combustion) then that's a bigger change. But I'll probably do that as part of a separate PR.